doc/source/reference/apis/metadata.md

```diff
@@ -22,7 +26,97 @@ this feature graduates. If you have any comments or suggestion please open the i
 
 ## Model Metadata (incubating)
 
-With Seldon you can easily define metadata for your models that is compatible with [kfserving dataplane proposal](https://github.com/kubeflow/kfserving/blob/master/docs/predict-api/v2/required_api.md#model-metadata) specification:
+With Seldon you can easily add metadata to your models.
+You can add metadata using one of two formats:
+- `v1` format that closely correlate to the current structure of `SeldonMessage`
+- `v2` format that is future-proof and fully compatible with `kfserving` dataplane proposal
```

Comment:
Maybe we can add here the link that we had previously?

```markdown
[`kfserving` dataplane proposal`](https://github.com/kubeflow/kfserving/blob/master/docs/predict-api/v2/required_api.md#model-metadata)
```

Reply:
The link is still bellow but can add it here as well!